base/sim/FairPrimaryGenerator.h

```diff
@@ -52,6 +52,9 @@ class FairPrimaryGenerator : public TNamed
     /** Initialize the generater (if needed!)*/
     virtual Bool_t Init();
 
+    /** Finalize the generater (if needed!)*/
+    virtual Bool_t Finish();
```

Comment:
This one always returns `kTRUE`. Are people expected to check the return value?
If not, let's return `void`?

Are we expecting this function to be overriden at some point? If not, we could keep it non-virtual to start with, maybe?